wordpress-mobile/WordPress-Android
# Issue/12251 add site title qs task

Fixes #12251

This PR depends on the [following](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1622) Flux-C PR. It needs to be merged first, then Flux-C reference updated.

In this PR we added a new Quick Start action for updating a Site Title from My Site screen.

[![Image from Gyazo](https://i.gyazo.com/4e1cdfa6eec0accffc168af049be6514.gif)](https://gyazo.com/4e1cdfa6eec0accffc168af049be6514)

[![Image from Gyazo](https://i.gyazo.com/eca1b6843e7291a7420cf852a172e4b1.png)](https://gyazo.com/eca1b6843e7291a7420cf852a172e4b1)

To test:
- Create a new site and start Quick Start.
- Navigate to `Customize Your Site` QS category and select `Set your site title` task.
- Notice that the QS Snackbar mentions site title, and the focus point is displayed at site title.
- Tap on the site title, and make sure the task is marked as completed. You don't need to change the actual title to complete the task.

PR submission checklist:
